test/jdk/java/lang/invoke/VarHandles/VarHandleTestAccessBoolean.java

```diff
@@ -280,6 +280,8 @@ public Object[][] accessTestCaseProvider() throws Exception {
         return cases.stream().map(tc -> new Object[]{tc.toString(), tc}).toArray(Object[][]::new);
     }
 
+    // CompileThresholdScaling is set to 0.1 and ITERS to 2000 to reduce
+    // the test execution
```

Comment:
Wouldn't this be clearer as an `@comment` in the test header prior to the `@run` statements that use the variables?

Reply:
Good suggestion.  Updated.